msteams-platform/resources/dev-preview/developer-preview-intro.md

```diff
@@ -25,6 +25,9 @@ Using an app that contains developer preview features may cause clients that hav
 
 ### On a desktop or web client
 
+> [!NOTE]
+> If your tenant is enrolled for *Microsoft 365 Targeted Releases* developer preview is automatically enabled and you will not see the option to switch to developer preview.
```

Comment:
Let's add a link to:

https://learn.microsoft.com/microsoft-365/admin/manage/release-options-in-office-365